config/urls/case_category_url.php

```diff
@@ -26,7 +26,7 @@
   ],
   'civicrm/ajax/rest' => [
     'url_type' => CaseCategoryFromUrl::AJAX_TYPE_URL,
-    'param' => 'case_type_id.case_type_category',
+    'param' => 'case_type_category',
```

Comment:
Not sure if we should change this.  Ajax calls from the FE to fetch case category for a case are usually like this

```php

CRM.api3('Case', 'get', {
  "sequential": 1,
  "case_type_id.case_type_category": 2
}).then(function(result) {
  // do something with result
}, function(error) {
  // oops
});

```

Hence the reason  to target this parameter, 
Changing this will definately lead to unexpected results for API calls for different case categories

Reply:
Not applicable after refactoring.